Задание 2/Program.cs

```diff
@@ -0,0 +1,35 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Задание_2номер_9
+{
+    class Program
+    {
+        static void Main()
+        {
+            Console.WriteLine("Введите длину стороны треугольника и прилежащие к ней углы");
```

Comment:
Лучше назвать пользователю единицу измерения углов
```suggestion
            Console.WriteLine("Введите длину стороны треугольника и прилежащие к ней углы в градусах");
```